diff --git a/py/zaber_motion/zaber_motion/bindings.py b/py/zaber_motion/zaber_motion/bindings.py
index b8ad68d09..9a6f61e89 100644
--- a/py/zaber_motion/zaber_motion/bindings.py
+++ b/py/zaber_motion/zaber_motion/bindings.py
@@ -1,23 +1,13 @@
 from ctypes import c_void_p, c_int, c_int64, c_uint8, CFUNCTYPE
+from ctypes import cdll
+import ctypes.util
 from typing import Any
 import platform
 import sys
-import importlib
-from .version import __version__
 
 
 def _load_library() -> Any:
-    is64bit = sys.maxsize > 2**32
     os_system = platform.system().lower()
-    os_machine = platform.machine().lower()
-
-    if os_system == "darwin":
-        arch = "uni"
-    else:
-        if os_machine.startswith("aarch64") or os_machine.startswith("arm"):
-            arch = "arm64" if is64bit else "arm"
-        else:
-            arch = "amd64" if is64bit else "386"
 
     ext = ""
     if os_system == "linux":
@@ -25,15 +15,12 @@ def _load_library() -> Any:
     if os_system == "darwin":
         ext = ".dylib"
 
-    lib_name = "zaber-motion-lib-{}-{}{}".format(os_system, arch, ext)
-    module_name = "zaber_motion_bindings_{}".format(os_system)
-    binding_module = importlib.import_module(module_name)
-    if __version__ != binding_module.__version__:
-        raise RuntimeError((
-            f"The dependent module {module_name} is not compatible with "
-            f"zaber_motion ({__version__} != {binding_module.__version__}). "
-            f"Please install the correct version {module_name}=={__version__}."))
-    return binding_module.load_library(lib_name)
+    lib_name = "zaber-motion-lib" + ext
+    # On windows, since Python 3.8 you need an extra call to find_library
+    # https://github.com/beetbox/pyacoustid/issues/54
+    if sys.platform == 'win32':
+        lib_name = ctypes.util.find_library(lib_name)
+    return cdll.LoadLibrary(lib_name)
 
 
 lib = _load_library()
