From cb3bc6aac1e0d3c12f7f4a8d7d8becd143b3e3e1 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 8 Feb 2025 09:53:25 -0500
Subject: [PATCH] Do not install bindings the way tey do upstream

---
 py/zaber_motion/bindings.py | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/py/zaber_motion/bindings.py b/py/zaber_motion/bindings.py
index f13036a6d..a27295f2b 100644
--- a/py/zaber_motion/bindings.py
+++ b/py/zaber_motion/bindings.py
@@ -1,6 +1,5 @@
 from ctypes import c_void_p, c_int, c_int64, c_uint8, CFUNCTYPE, cdll
 from typing import Any
-from os import path
 import platform
 import sys
 
@@ -27,13 +26,7 @@ def _load_library() -> Any:
     else:
         raise ImportError(f"Unsupported operating system {os_system}")
 
-    lib_name = f"zaber-motion-core-{os_system}-{arch}{ext}"
-    lib_path = path.join(path.dirname(__file__), "..", "zaber_motion_bindings", lib_name)
-    lib_path = path.normpath(lib_path)
-
-    if not path.exists(lib_path):
-        lib_dir = path.dirname(lib_path)
-        raise ImportError(f"Could not find library {lib_name} at path {lib_dir}. Please contact Zaber support.")
+    lib_name = f"zaber-motion-lib{ext}"
     return cdll.LoadLibrary(lib_path)
 
 
-- 
2.43.0

